docs/user/openstack/connect_nodes_to_ipv6_network.md

```diff
@@ -0,0 +1,63 @@
+# Connect workers nodes and Pods to IPv6 network
+
+To connect your workers at the time of installation you can use [additionalNetworkIDs](https://github.com/openshift/installer/blob/master/docs/user/openstack/customization.md#additional-networks) parameter in the install config and set IPv6 network ID there:
+
+Example OpenShift install config:
+
+```yaml
+...
+compute:
+- name: worker
+  platform:
+    openstack:
+      additionalNetworkIDs:
+      - <ipv6_network_id>
+...
+```
+
+**Note**: To use Stateful IPv6 Networks, the arg `ip=dhcp,dhcpv6` needs to be included in the kernal args of the Worker nodes, otherwise the Nodes won't get an IPv6 address due to a [bug][dhcpv6-bug]. Use the [procedure][add-kernel-args] to add kernel argument to the Nodes.
+
+## Enable connectivity to the Pods
+
+To enable connectivity between Pods with additional Network on different Nodes, the Port security needs to be disabled for the IPv6 Port of the Server. This way it's possible to avoid adding an allowed-address-pairs with an IP and MAC address in the Server's IPv6 Port whenever a new Pod gets created.
+
+```sh
+openstack port set --no-security-group --disable-port-security <worker-ipv6-port>
+```
+
+## Add IPv6 connectivity to Pods
+
+Create a file named `network.yaml` and specify [the desired CNI config][configuring-an-additional-network]. Here is an example of CNI config used for slaac address mode with macvlan:
+
+```yaml
+spec:
+  additionalNetworks:
+  - name: ipv6
+    namespace: ipv6
+    rawCNIConfig: '{ "cniVersion": "0.3.1", "name": "ipv6", "type": "macvlan", "master": "ens4"}'
```

Comment:
I don't think `ens4` is set in stone, we should probably tell user that this will be different when there are more additionalNetworkIDs or when a different kernel driver is used (there were iface naming issues with PowerPC and Kuryr).

Reply:
Indeed. I just updated with it, thanks